src/compositions/DisasterPosts/DisasterPosts.css

```diff
@@ -50,22 +50,58 @@
 
 .DisasterPosts .modal-details {
     display: block;
-    position: relative;
-    margin: auto;
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
     z-index: 6;
     width: 60%;
-    height: fit-content;
+    overflow-y: auto;
     background-color: white;
     text-align: center;
 }
 
-.DisasterPosts .modal-details .map {
+.DisasterPosts .modal-details .modal-col {
```

Comment:
can you do these details with CSS Grid instead of display inline-block and float?